test/e2e/media.spec.ts

```diff
@@ -54,14 +54,8 @@ describe('Media Metadata endpoints', () => {
       expect(doc).toHaveProperty('title', 'Homeland');
       expect(doc).toHaveProperty('startYear', '2011');
 
-      // similar searches should return the same series metadata document
```

Comment:
So does this mean this doesn't pass now with this change?

Reply:
Yeah. It's up to the client to pass a series name to the API

Reply:
The client has its own set of tests for its parsing of filenames btw https://raw.githubusercontent.com/UniversalMediaServer/UniversalMediaServer/master/src/test/resources/net/pms/util/prettified_filenames_metadata.json